os-config.tpl.yml

```diff
@@ -161,6 +161,19 @@ rancher:
       read_only: true
       volumes:
       - /var/lib/docker:/var/lib/docker
+    logrotate:
+      image: {{.OS_REPO}}/os-logrotate:{{.VERSION}}{{.SUFFIX}}
+      command: /usr/sbin/logrotate -v /etc/logrotate.conf
+      labels:
+        io.rancher.os.createonly: "true"
+        io.rancher.os.scope: system
+        io.rancher.os.before: system-cron
```

Comment:
I noticed that on some boots, cron would not pick up the logrotate container's label, which I'm guessing was due to a start up race condition or sorts. I added this to make sure the container is created before crond starts.

Example of missing detection:
```
[root@rancher ~]# system-docker logs system-cron
time="2017-07-06T17:40:49Z" level=info msg="Starting Cron"
time="2017-07-06T17:40:49Z" level=info msg="Scanning for container cron entries"
```